src/main/java/com/github/manolo8/darkbot/config/Config.java

```diff
@@ -182,9 +183,17 @@
         public @Option boolean DEV_STUFF = false;
         public @Option @Editor(JListField.class) @Options(ApiSupplier.class) int API = 2;
         public @Option boolean FULLY_HIDE_API = true;
+        public @Option boolean SAVE_MAIN_GUI_POS_AND_SIZE = false;
```

Comment:
This is way too long. 
```suggestion
        public @Option boolean SAVE_GUI_POS = false;
```